src/basic/macro.h

```diff
@@ -29,6 +29,19 @@
 #define _alignptr_ __attribute__((__aligned__(sizeof(void*))))
 #define _warn_unused_result_ __attribute__((__warn_unused_result__))
 
+#if defined(BUILT_WITH_COVERAGE) && BUILT_WITH_COVERAGE
```

Comment:
Please drop the `defined(BUILT_WITH_COVERAGE) && `.

The big advantage of using `.set10()` instead of `.set()` is that the variable is always defined and we can catch typos. In ancient times autoconf would only do `#define WITH_FOO`, so the usual check was `#ifdef WITH_FOO`. The problem with that is that it's fairly easy to remove the definition without getting any errors about remaining uses. When we dropped autoconf support finally converted to use `.set10()` and `-Werror=undef` there were a few places with dead code because of messed up `ifdef`s.

Reply:
Ah, I forgot it there from the previous version, where the code was located in `macro-fundamental.h` (and `BUILT_WITH_COVERAGE` was not defined when building EFI stuff).